content/posts/2023/reana-0.9.2.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "REANA 0.9.2 is released"
+date: 2023-12-18T09:00:00+01:00
+---
+
+REANA 0.9.2 has just been released! This is a minor update that removes the limit on the number of restarts of a workflow, brings some more performance improvements, provides initial support for ARM platforms, and more!
+
+<!--more-->
+
+## What's new for the users?
+
+### Increasing workflow restart limit
+
+REANA already offers the possibility to restart a workflow, that is to execute a new workflow whose workspace is shared with the previous one. This comes in handy when developing a new analysis, for example to test small changes to the code or to fix some issues.
+
+Up until now, a workflow could be restarted only up to nine times. This limitation has been removed with REANA 0.9.2 and it is now possible to restart workflows as many times as needed.
+
+```console
+$ reana-client restart -w reana-demo-root6-roofit.1.9
+==> SUCCESS: reana-demo-root6-roofit.1.10 is pending
+```
+
+### Performance improvements
+
+Similarly to the previous release, REANA 0.9.2 brings some more performance improvements.
+In particular, the usage of the database has been optimised in order to reduce the time needed to perform some of the most common operations, including listing and filtering workflows and jobs.
+
+These improvements can be quite noticeable, even more so when users have executed tens of thousands of workflows and jobs.
+For example, we have measured a performance improvement of around 18x (18 times, not percent!) when listing all the details of a thousand workflows with `reana-client-go`.
```

Comment:
These improvements can be quite noticeable for users with many workflows. For example, we have measured a speed-up of the `reana-client list` query to be 18 times faster for a user having ten thousands of workflow runs!

_(cannot use Go client since it's not officially released yet)_